File: core/client/game/src/commonGame/logic/union/PlayerUnion.cs
using System;
using ShineEngine;

/// <summary>
/// 玩家工会
/// </summary>
public class PlayerUnion:PlayerRoleGroup
{
	/** 创建公会简版数据 */
	protected override RoleGroupSimpleData toCreateRoleGroupSimpleData()
	{
		return new UnionSimpleData();
	}
}

File: core/server/clientProject/commonClient/src/main/java/com/home/commonClient/logic/union/PlayerUnion.java
package com.home.commonClient.logic.union;

import com.home.commonBase.data.social.roleGroup.RoleGroupSimpleData;
import com.home.commonBase.data.social.union.UnionSimpleData;
import com.home.commonClient.logic.func.PlayerRoleGroup;

public class PlayerUnion extends PlayerRoleGroup
{
	/** 创建公会简版数据 */
	@Override
	protected RoleGroupSimpleData toCreateRoleGroupSimpleData()
	{
		return new UnionSimpleData();
	}
}
